daprdocs/content/en/operations/extending/developing-grpc-components.md

```diff
@@ -0,0 +1,339 @@
+---
+type: docs
+title: "Developing gRPC-based Pluggable Components"
+linkTitle: "Developing gRPC-based Pluggable Components"
+weight: 250
+description: "Extending dapr with external gRPC-based components"
+---
+
+[gRPC-based](https://grpc.io/) Dapr components are typically run as containers or processes that communicate with the dapr main process via [Unix Domain Sockets](https://en.wikipedia.org/wiki/Unix_domain_socket).
```

Comment:
It would be better to create context to state what gRPC Dapr components _are_ instead of saying _how_ they are typically used. As this is the first document in this series, this document should provide context of what is to follow if the reader choose to embark in this reading journey - or link to some place that would provide such.

Suggestion: Use reference links to avoid repeating the main GRPC link. Not really a big deal as you only link it twice but it would allow you to easily turn any textual reference to gRPC into a links.